rsmtool/notebooks/explanations/data_description.ipynb

```diff
@@ -39,17 +39,21 @@
    "id": "43aeb967",
    "metadata": {},
    "source": [
-    "### Background Distribution"
+    "### Background Data"
    ]
   },
   {
    "cell_type": "markdown",
    "id": "6171e672",
    "metadata": {},
    "source": [
-    "The background distribution is responsible for generating the base value of your shap explanation. The base value represents the mean model prediction in the data that was passed as background.\n",
+    "The background data is responsible for generating the *base value* of your SHAP explanations. Specifically, this is the average model prediction in the background data passed to `rsmexplain`.\n",
     "\n",
-    "By default, rsmexplain generates a kmeans-clustered representation of your background sample. The size of which you can specify in the \"background_kmeans_size\" parameter. Generally a kmeans sample of size 100 or higher is sufficient to generate an accurate base value.\n"
+    "SHAP computes feature contributions by replacing feature values in the data you wish to explain with values sampled from the background set and measuring changes in the prediction. Therefore, The background data is crucial in helping us understand the impact of a feature moving from its 'baseline' or 'average' state to the current state. This movement is what the SHAP values shown in the report below quantify. This means that as long as **sufficiently large and diverse** background data is used, the SHAP values for individual examples in the explain data can be considered reliable.\n",
+    "\n",
+    "However, running `rsmexplain` on very large background datasets can be computationally expensive and time-consuming. One trick we can use is to use k-means clustering to \"summarize\" and reduce the size of the background dataset without losing too much information. To do this, we run a k-means clustering algorithm on the background dataset which yields `k` clusters of feature values, each represented by a \"centroid\" (the average of all data points in the cluster). These centroids are then used to compute the SHAP values, i.e., when we \"omit\" a feature and go to replace its value, we do so by sampling from from one of the centroids rather than the original dataset. \n",
```

Comment:
```suggestion
    "However, running `rsmexplain` on very large background datasets can be computationally expensive and time-consuming. One trick we can use is to use k-means clustering to \"summarize\" and reduce the size of the background dataset without losing too much information. To do this, we run a k-means clustering algorithm on the background dataset which yields `k` clusters of feature values, each represented by a \"centroid\" (the average of all data points in the cluster). These centroids are then used to compute the SHAP values, i.e., when we \"omit\" a feature and go to replace its value, we do so by sampling from one of the centroids rather than the original dataset. \n",
```